here-naksha-lib-model/src/commonMain/kotlin/naksha/model/request/Write.kt

```diff
@@ -171,7 +171,7 @@ open class Write : AnyObject() {
         this.collectionId = collectionId
         this.op = WriteOp.UPDATE
         this.id = feature.id
-        this.version = if (atomic) feature.properties.xyz.version else null
+        this.version = if (atomic) feature.properties.xyz.txn else null
```

Comment:
do you think we should change `version` in Write request as well?

Reply:
Hard to say right now,.
If XYZ confirms that `version` should be 64-bit, we can stick with `version`, because its only an MOM error that anyway need to be fixed (so its just a temporary workaround to externally name it `txn`). The reason for the version after all was compatibility with IMS.
I'm open to generally introduce `txn` back as our internal versioning (then we should rename `Version` back into `TransactionNumber`). We could add an _optional_ version, that is just an integer and matches global version in Storage-API, and version in IMS, if they really use 32-bit integers.
So: I asked Dimitar, if the documentation and MOM is wrong, if it is I propose to stick with `Version`, so for now I would say: Do not rename. However, should Dimitar clarify that `version` in IMS is a 32-bit value, we need to permanently switch to `txn`.